packages/clay-alert/src/ClayAlert.soy

```diff
@@ -37,43 +37,79 @@
 	{/let}
 
 	<div {$attributes}>
-		{if $closeable}
-			{call ClayButton.render}
-				{param ariaLabel: 'Close' /}
-				{param elementClasses: 'close' /}
-				{param events: [ 'click': $_handleCloseClick ] /}
-				{param icon: [
-					'alignment': 'right',
-					'spritemap': $spritemap,
-					'symbol': 'times'
-					] /}
-				{param style: 'unstyled' /}
-				{param type: 'button' /}
+		{if $type == 'fluid'}
+			<div class="container">
+				{call .content}
```

Comment:
Due is the same content I'd say to extract this `call` out of the `{if}` and create a variable `$content`

```
	{let $content kind="html"}
		{call .content}
			{param _handleCloseClick: $_handleCloseClick /}
			{param closeable: $closeable /}
			{param message: $message /}
			{param spritemap: $spritemap /}
			{param style: $style /}
			{param title: $title /}
		{/call}
	{/let}

	{if $type == 'fluid'}
		<div class="container">
			{$content}
		</div>
	{else}
		{$content}
	{/if}
```